src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,87 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.util.Arrays;
+import java.util.Objects;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    private String email;
```

Comment:
To ensure the `email` field is unique in the database, consider adding a unique constraint annotation, such as `@Column(unique = true)`, to the `email` field. This will enforce the uniqueness of email addresses at the database level.